src/store/project.ts

```diff
@@ -357,6 +360,8 @@ export const projectStore = createPartialStore<ProjectStoreTypes>({
           const message = (() => {
             if (typeof err === "string") return err;
             if (!(err instanceof Error)) return "エラーが発生しました。";
+            if (err instanceof ResultError && err.code === "ENOENT")
+              return "プロジェクトファイルが見付かりませんでした。ファイルが移動、または削除された可能性があります。";
```

Comment:
```suggestion
              return "プロジェクトファイルが見つかりませんでした。ファイルが移動、または削除された可能性があります。";
```

横から失礼します.　下に揃えるなら “見つかる/見つからない” かも…？

https://github.com/VOICEVOX/voicevox/blob/6ff3a95af34385619a47492c8406b7ef4e6c1009/src/components/EngineManageDialog.vue#L470

Reply:
確かに。ありがとうございます。